src/app/Question.php

```diff
@@ -6,8 +6,10 @@
 
 class Question extends Model
 {
+    public $timestamps = false;
+
     public function choices()
     {
-        return $this->hasMany('App\Choice');
+        return $this->hasMany('App\Choice')->where('hide', 0);
```

Comment:
choice側のglobal scopeで考慮すべきことかなと思いました。
hide ===  1の項目がほしいことが少ないと思うのでその時に指定してあげるほうが公開?非公開?を意識しない作りになるのかなと思います
```suggestion
        return $this->hasMany('App\Choice');
```